bbot/modules/git_clone.py

```diff
@@ -39,9 +39,7 @@ async def handle_event(self, event):
         repo_path = await self.clone_git_repository(repo_url)
         if repo_path:
             self.verbose(f"Cloned {repo_url} to {repo_path}")
-            codebase_event = self.make_event(
-                {"path": str(repo_path)}, "FILESYSTEM", tags=["git", "folder"], source=event
-            )
+            codebase_event = self.make_event({"path": str(repo_path)}, "FILESYSTEM", tags=["git"], source=event)
```

Comment:
The removal of the folder tag here will stop the unstructured module from crawling the folder